app/models/alternate-identifier.js

```diff
@@ -5,7 +5,7 @@ import { computed } from '@ember/object';
 import { isBlank } from '@ember/utils';
 
```

Comment:
I think the model attribute needs to be changed here https://github.com/datacite/bracco/blob/dc7f5ee/app/models/alternate-identifier.js#L12

Reply:
Thanks.